src/lib.rs

```diff
@@ -63,7 +63,9 @@ macro_rules! report_error {
     ($error:expr, $($args:tt)*) => {
         {
           let kind = ::std::any::type_name_of_val(&$error);
-          $crate::tracing::error!(error.kind = kind, error = &$error as &dyn ::std::error::Error, $($args)+)
+          let stack = error_report::Report::new(&error);
```

Comment:
@damjack this is not a public crate, it's a crate in `intermediaries-crates` 😅 
I don't think prima_tracing can pull it